pkg/server/methods/executeCommand.go

```diff
@@ -12,6 +13,14 @@ import (
 	"go.lsp.dev/uri"
 )
 
+type BadArgumentError struct {
+	ArgName string
+}
+
+func (err BadArgumentError) Error() string {
+	return fmt.Sprintf("wrong argument: %s", err.ArgName)
+}
+
```

Comment:
Instead of creating a new error type here, can we make use of the built-in [`ErrInvalidParams`](https://pkg.go.dev/go.lsp.dev/jsonrpc2#pkg-variables)? Or maybe use the constant error code for invalid params, but customize the message as you do?

```go
jsonrpc2.NewError(jsonrpc2.InvalidParams, "invalid method parameter: token")
```